kafka-observ-lib/signals/topic.libsonnet

```diff
@@ -90,6 +101,10 @@ function(this)
             aggKeepLabels: ['topic', 'partition'],
             expr: 'kafka_log_log_logstartoffset{%(queriesSelector)s}',
           },
+          bitnami: {
+            aggKeepLabels: ['topic', 'partition'],
+            expr: 'kafka_log_log_logstartoffset{%(queriesSelector)s}',
```

Comment:
All the other metrics so far for bitnami end in `_count`. Is this one different?

Reply:
almost. There are some tiny differences sometimes, like _total_ in the middle of one of the metrics.